### PR TITLE
add owner for simpleprometheus

### LINKED ISSUE #11504: [receiver/simpleprometheus] component is unmaintained
The component has no code owner.

If you're interested in maintaining the component, please respond on this ticket and create a PR to add yourself as a code owner.

Open issues for the component:
- [x] #7180
- [x] #9571

### Details

Code owners for this component have explicitly stated they will no longer maintain the code
or have been unresponsive to issues/pull requests.

- [x] Wait 6 weeks for the code owner to provide feedback, either here or in the linked issue
- [x] Attempt to recruit new code owners, potentially from the community
- [x] On Jun-24-2026, the component was labeled as unmaintained
- [ ] On Dec-24-2026, 6 months from now, the component will be removed

### PR DESCRIPTION
**Pull request by fatsheep9146**
Signed-off-by: Ziqi Zhao <zhaoziqi9146@gmail.com>

related issue: https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/9571#issuecomment-1164553895

Resolves https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/11504

### CODE REVIEW

**Comment by fatsheep9146** [comment]
ping @codeboten , please help review this, thanks.

**Review by codeboten — COMMENTED** [review]
Thanks @fatsheep9146! Please update the changelog to remove the line about the simpleprometheus receiver being unmaintained

**Comment by fatsheep9146** [comment]
> Thanks @fatsheep9146! Please update the changelog to remove the line about the simpleprometheus receiver being unmaintained

Done

File: CHANGELOG.md [hunk 1]
```diff
@@ -21,7 +21,7 @@
 
 ### Unmaintained components
 
-- `simpleprometheusreceiver`(#11133)
+- `tanzuobservabilityexporter`: remove calls to deprecated `NewProxySender` methods. (#11510)
```
Comment (dmitryax) [inline CHANGELOG.md:24]: Why is this line added?

**Review by dmitryax — COMMENTED** [review]
Please remove log line added in this PR https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/11133

File: CHANGELOG.md (hunk 1 above)
Comment (fatsheep9146) [reply CHANGELOG.md:24]: Sorry, I mistakely add it back when I resolved the conflicts. I'll remove it

Comment (fatsheep9146) [reply CHANGELOG.md]: Done

**Comment by dmitryax** [comment]
Running the receiver will stil throw a unmaintained warning, please remove that

File: receiver/simpleprometheusreceiver/README.md [hunk 2]
```diff
@@ -72,5 +72,4 @@ Example:
 The full list of settings exposed for this receiver are documented [here](./config.go)
 with detailed sample configurations [here](./testdata/config.yaml).
```
Comment (codeboten) [inline receiver/simpleprometheusreceiver/README.md:74]: This will need a link for `beta`

**Review by codeboten — COMMENTED** [review]
Just one comment, thanks for taking this on!

File: receiver/simpleprometheusreceiver/README.md (hunk 2 above)
Comment (fatsheep9146) [reply receiver/simpleprometheusreceiver/README.md:74]: Done

**Comment by dmitryax** [comment]
Please run `make gotidy`

**Review by dmitryax — APPROVED** [review]

**Comment by fatsheep9146** [comment]
ping @codeboten I think that this pr is ready to merge. :)

**Review by mx-psi — APPROVED** [review]
